### ISSUE TITLE
Error code 9999

### ISSUE THREAD

**Issue by niyonx** [mevdschee/php-crud-api#876]
Hi, I have received the error 

```
{ "code": 9999, "message": "Tqdev\PhpCrudApi\Record\Condition\AndCondition::fromArray(): Return value must be of type Tqdev\PhpCrudApi\Record\Condition\Condition, Tqdev\PhpCrudApi\Record\Condition\NoCondition returned" }
```

on every get request. I have multiple tables and each one was working. I have changed the database and still received this error.

I had a backup of the code and used the same database, it works perfectly. Does anyone know when this error occurs?

**Comment by mevdschee** [comment]
Please help us reproduce the problem.

**Comment by mevdschee** [comment]
I'm also warning you that if you keep opening issues without reproduction steps I will block you from this project's issue tracker, see #875

**Comment by niyonx** [comment]
hey @mevdschee I apologize. I really don't know how to reproduce it unfortunately and it seemed to be a general error on every request...

**Comment by mevdschee** [comment]
Can you share a database create

**Comment by mevdschee** [comment]
Can you share your configuration and your sql (table) create statement(s) and the request you are doing to trigger this error?

**Comment by mevdschee** [comment]
There is no way anyone can help you without more information from your side. If you don't want to share publicly you may sent me an email. Kind regards, Maurits

**Comment by mevdschee** [comment]
Did you run the tests? Did they succeed? What PHP version are you running? What operating system are you on?

**Comment by mevdschee** [comment]
I have just released v2.14.0, can you test whether or not this version still has the problem you encounter?

**Comment by niyonx** [comment]
Hi @mevdschee, I have re-tested the exact same setup as the one throwing me errors before, everything behaves properly now.

Around the same time I had a cloudflare issue which was not returning my requests properly. I think it was what was causing the issue as I did a diff, no code change and the database was good.

Thank you for your help!
